legend-engine-language-pure-store-serviceStore/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/ServiceStoreParserGrammar.g4

```diff
@@ -0,0 +1,128 @@
+parser grammar ServiceStoreParserGrammar;
+
+import M3ParserGrammar;
+
+options
+{
+    tokenVocab = ServiceStoreLexerGrammar;
+}
+
+
+// -------------------------------------- IDENTIFIER --------------------------------------
+
+identifier:                                 VALID_STRING | STRING
+                                            | ALL | LET | ALL_VERSIONS | ALL_VERSIONS_IN_RANGE      // from M3Parser
+                                            | SERVICE_STORE
+                                            | DESCRIPTION | SERVICE_GROUP | SERVICE
+                                            | PATH | REQUEST_BODY | METHOD | PARAMETERS | RESPONSE | SECURITY_SCHEME
+                                            | LOCATION | STYLE | EXPLODE | ENUM
+                                            | SERVICE_MAPPING | PARAM_MAPPING | SERVICE_REFERENCE
+;
+
+// -------------------------------------- DEFINITION --------------------------------------
+
+definition:                                 (serviceStore)*
+                                            EOF
+;
+serviceStore:                               SERVICE_STORE qualifiedName
+                                                PAREN_OPEN
```

Comment:
@pierredebelen Can you please opinion if we should have '(' or '{' as starting for Service store
```
ServiceStore dummy
(
    elements ...
)

vs

ServiceStore dummy
{
    elements ...
}
```

Currently I have '(' to keep it consistent with relation store definition